lib/pages/home.dart

```diff
@@ -421,22 +421,12 @@ class _DesktopCategoryItem extends StatelessWidget {
                 color: colorScheme.background,
               ),
               Flexible(
-                // Remove ListView top padding as it is already accounted for.
-                child: MediaQuery.removePadding(
-                  removeTop: true,
-                  context: context,
-                  child: ListView(
-                    // Makes integration tests possible.
-                    key: ValueKey('${category.name}DemoList'),
-                    children: [
-                      const SizedBox(height: 12),
```

Comment:
I wonder why we had this to begin with. It doesn't look that great so makes sense to remove it.

Reply:
It was in the designs, but I agree that it looks better without it